### PR TITLE
Added the file "joerup2004"

### PR DESCRIPTION
**Pull request by joerup2004**
Adds the file joerup2004

### CODE REVIEW

File: Participants/joerup2004 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Name: Joe Rupertus
+Class year: 2026
```
Comment (bnehoran) [inline Participants/joerup2004:2]: Nice class year!

File: Participants/joerup2004 [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+Odd fact: I am also a raccoon
```
Comment (bnehoran) [inline Participants/joerup2004:3]: Cool!

**Review by bnehoran — CHANGES_REQUESTED** [review]
Add an exclamation point to the raccoon statement.

**Comment by joerup2004** [comment]
Made the changes

**Review by bnehoran — APPROVED** [review]
Looks good!

**Pull request merged**